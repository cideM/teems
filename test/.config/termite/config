[options]
scroll_on_output = false
scroll_on_keystroke = true
audible_bell = false
mouse_autohide = false
allow_bold = true
dynamic_title = true
urgent_on_bell = true
clickable_url = true
font = Iosevka Term 14 
border_width = 0
scrollback_lines = 10000
search_wrap = true
#icon_name = terminal
#geometry = 640x480

# "system", "on" or "off"
cursor_blink = system

# "block", "underline" or "ibeam"
cursor_shape = block

# $BROWSER is used by default if set, with xdg-open as a fallback
#browser = xdg-open

# set size hints for the window
#size_hints = false

# Hide links that are no longer valid in url select overlay mode
filter_unmatched_urls = true

# emit escape sequences for extra modified keys
#modify_other_keys = false

# Dracula, from the Xresources theme linked on its main repo
# https://github.com/dracula/xresources/blob/fa6a33dfc6f270b643fee32713a90225f9b0d4fc/Xresources
[colors]

# special
foreground = #FFFFFF
foreground_bold = #FFFFFF
cursor = 
background = #000000

# black
color0 = #FFFF00
color8  = #4D4D4D

# red
color1  = #FF5555
color9  = #FF6E67

# green
color2  = #50FA7B
color10 = #5AF78E

# yellow
color3  = #F1FA8C
color11 = #F4F99D

# blue
color4  = #BD93F9
color12 = #CAA9FA

# magenta
color5  = #FF79C6
color13 = #FF92D0

# cyan
color6  = #8BE9FD
color14 = #9AEDFE

# white
color7  = #BFBFBF
color15 = #E6E6E6
