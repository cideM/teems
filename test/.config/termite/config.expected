[options]
scroll_on_output = false
scroll_on_keystroke = true
audible_bell = false
mouse_autohide = false
allow_bold = true
dynamic_title = true
urgent_on_bell = true
clickable_url = true
font = Iosevka Term 14 
border_width = 0
scrollback_lines = 10000
search_wrap = true
#icon_name = terminal
#geometry = 640x480

# "system", "on" or "off"
cursor_blink = system

# "block", "underline" or "ibeam"
cursor_shape = block

# $BROWSER is used by default if set, with xdg-open as a fallback
#browser = xdg-open

# set size hints for the window
#size_hints = false

# Hide links that are no longer valid in url select overlay mode
filter_unmatched_urls = true

# emit escape sequences for extra modified keys
#modify_other_keys = false

# Dracula, from the Xresources theme linked on its main repo
# https://github.com/dracula/xresources/blob/fa6a33dfc6f270b643fee32713a90225f9b0d4fc/Xresources
[colors]

# special
foreground = #FFFFFF
foreground_bold = #FFFFFF
cursor = 
background = #000000

# black
color0 = #FFFF00
color8 = #FFFF08

# red
color1 = #FFFF01
color9 = #FFFF09

# green
color2 = #FFFF02
color10 = #FFFF10

# yellow
color3 = #FFFF03
color11 = #FFFF11

# blue
color4 = #FFFF04
color12 = #FFFF12

# magenta
color5 = #FFFF05
color13 = #FFFF13

# cyan
color6 = #FFFF06
color14 = #FFFF14

# white
color7 = #FFFF07
color15 = #FFFF15
